google/cloud/aiplatform/featurestore/entity_type.py

```diff
@@ -1146,6 +1149,148 @@ def ingest_from_gcs(
             request_metadata=request_metadata,
         )
 
+    def ingest_from_df(
+        self,
+        feature_ids: List[str],
+        feature_time: Union[str, datetime.datetime],
+        df_source: "pd.DataFrame",  # noqa: F821 - skip check for undefined name 'pd'
+        feature_source_fields: Optional[Dict[str, str]] = None,
+        entity_id_field: Optional[str] = None,
+        disable_online_serving: Optional[bool] = None,
```

Comment:
Given ingest data from dataframe should not be too big (less than few Gb in most cases, I am thinking we probably want to hide these 2 options (disable_online_serving and worker_count) for simplicity. It also easier to change to use streaming ingestion in the future (which is faster and simpler). Wdyt?

Reply:
It maybe easier to keep the optional field in case user want to use it (unless these for sure will not be needed?) than have to add it back later. Also, we have different methods for online serve and batch serve, it may make more sense to have both online ingest and batch ingest. Wdyt?

Reply:
I'm not sure if we will provide dual implementation (using batch vs streaming ingestion API). I'm leaning toward just using streaming ingestion as it's both simpler to implement and faster for notebook use-case. With that API, disable_online_serving and number of workers is not relevant. I feel it would be more complex to deprecate it than add it back when really need it.

I vote for leave it out for now, and we can revisit when streaming ingestion is ready for this function.

Reply:
Sounds good. Removed the two args.